Bottom: 30b6852de4312fafcd149d6affabe8716881a271
Top:    08ac188d9a8dee251df9834f140e22968612b1a7
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-31 12:12:39 +0800

Refresh of vhost_net-basic-batch

---

diff --git a/samples/bpf/xdp1_kern.c b/samples/bpf/xdp1_kern.c
index a4b89d3..2197421 100644
--- a/samples/bpf/xdp1_kern.c
+++ b/samples/bpf/xdp1_kern.c
@@ -51,8 +51,6 @@ int xdp_prog1(struct xdp_md *ctx)
 	u64 nh_off;
 	u32 ipproto;
 
-	return rc;
-
 	nh_off = sizeof(*eth);
 	if (data + nh_off > data_end)
 		return rc;
